assets/js/wc-gateway-ppec-smart-payment-buttons.js

```diff
@@ -94,11 +67,11 @@
 					.on( 'disable', actions.disable );
 			},
 
-			payment: function() {
+			createOrder: function() {
 				// Clear any errors from previous attempt.
 				$( '.woocommerce-error', selector ).remove();
 
-				return new paypal.Promise( function( resolve, reject ) {
+				return new Promise( function( resolve, reject ) {
```

Comment:
I don't think this will work on Internet Explorer 11 p7bbVw-3nP-p2

Reply:
Same as https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/pull/622#discussion_r330920096; PayPal is deprecating `paypal.Promise`. See their [documentation](https://developer.paypal.com/docs/checkout/reference/upgrade-integration/#6-fix-deprecations).